typing/ctype.ml

```diff
@@ -1828,6 +1836,20 @@ let subst env level priv abbrev oty params args body =
     undo_abbrev ();
     raise Cannot_subst
 
+let jkind_subst env level params args jkind =
```

Comment:
I think @lpw25 should review this function.

Reply:
LGTM